infra-templates/windows-healthcheck/0/rancher-compose.yml

```diff
@@ -0,0 +1,11 @@
+.catalog:
+    name: "Healthcheck Service"
+    version: "v0.2.0"
+    description: "Rancher healthcheck service"
```

Comment:
Yes, good idea. I'm considering adding that constraint to all of the `windows-*` templates, thoughts?